codeforamerica/vita-min
# TBE-71 JMeter tests

## [TBE-71](https://codeforamerica.atlassian.net/browse/TBE-71)
## Is PM acceptance required?
## What was done?
- Created JMeter Tests and bash scripts for vita-min
- Test scripts try to mimic real usage as found in datadog
- Tests may be run against dev or staging
## How to test?
- See the README.md for details on Jmeter setup
- Run the command `jmeter_test/jmeter_test -t fyst_az_5_minute_stress_test`
## Screenshots (for visual changes)
### Local Results from my Laptop
![image](https://github.com/codeforamerica/vita-min/assets/17094895/91f52508-c611-45dd-bd1b-57cc909496a1)
[TBE-71]: https://codeforamerica.atlassian.net/browse/TBE-71?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
